samples/apps/copilot-chat-app/webapi/Config/AzureSpeechOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SemanticKernel.Service.Config;
+
+/// <summary>
+/// Configuration options for Azure speech recognition.
+/// </summary>
+public sealed class AzureSpeechOptions
+{
+    public const string PropertyName = "AzureSpeech";
+
+    /// <summary>
+    /// Region of the Azure speech service.
```

Comment:
Maybe this would be better:

```suggestion
    /// Location of the Azure speech service to use (e.g. "South Central US")
```